configs/sandpack-contents/component-theming/skeleton.js

```diff
@@ -0,0 +1,117 @@
+module.exports = {
+  App: `import {
+    Text,
+    Box,
+    Flex,
+    Skeleton,
+    Button,
+    IconButton,
+    useColorMode,
+  } from "@chakra-ui/react";
+  import { FaMoon, FaSun } from "react-icons/fa";
+  import { useState } from "react";
+  
+  export default function App() {
+    const [loading, setLoading] = useState(true);
+    const { toggleColorMode, colorMode } = useColorMode();
+  
+    return (
+      <Box pos="relative">
+        <Flex textAlign="center" gap={3} mt={3} direction="column">
+          <Box>
+            <Text>Default Skeleton</Text>
+            <Skeleton isLoaded={!loading} bg="orange" fadeDuration={1}>
+              <Text>Hidden Text</Text>
+            </Skeleton>
+          </Box>
+  
+          <Box>
+            <Text>With xl size</Text>
+            <Skeleton isLoaded={!loading} bg="orange" fadeDuration={1} size="xl">
+              <Text>Hidden Text</Text>
+            </Skeleton>
+          </Box>
+  
+          <Box>
+            <Text>With red variant</Text>
+            <Skeleton
+              isLoaded={!loading}
+              bg="orange"
+              fadeDuration={1}
+              variant="red"
+            >
+              <Text>Hidden Text</Text>
+            </Skeleton>
```

Comment:
Can we make the skeletons a little bit smaller when it comes to the width?

Reply:
On top of that, can we make one skeleton with different fadeDuration than 1?

Reply:
@aacevski, I've changed ```fadeDuration``` to 1, 1.5, and 2 for each of the skeletons respectively.